src/api/routes/transcripts.ts

```diff
@@ -52,26 +53,26 @@ const getMostRecentStartedAt = procedure
   return await db.Transcript.max("startedAt", { where: { groupId } });
 });
 
-const getNameMap = procedure
+const getUserMap = procedure
```

Comment:
Why renamed to user name? it should be consistent with zSummaryNameMap. So please use `getNameMap`.

Reply:
So there are two objects returning from the `getSummariesAndUserMap` function, one is { [handlebar:] [user name] }, which is what we initially called `Name Map`. And there is also another object  `{ [handlebar:] [user] }` which is referred to as `User Map` here now returned to the frontend in order to make the frontend code work more simply. `Name Map` is still kept to be passed to handlebars.js to compile